src/utils.js

```diff
@@ -711,5 +765,47 @@ go.utils = {
         return choices;
     },
 
+    save_identities: function(im, receiver, receiver_msisdn, father_msisdn, mother_msisdn, operator_id) {
```

Comment:
These params need docs, I don't know if they're meant to be id store UUIDs or strings of MSISDNs or integers etc.